package.json

```diff
@@ -71,12 +72,18 @@
     "./package.json": "./package.json"
   },
   "imports": {
-    "#cli/bin_path": "./cli/bin_path.js"
+    "#cli/paths": "./cli/_paths.js",
```

Comment:
Isn't this changing "public API" of the `rescript` npm package?

Reply:
subpath imports can only be called by the package itself. They are not exposed to other packages.